lib/components/user/terms-of-use-pane.js

```diff
@@ -46,11 +48,19 @@ const TermsOfUsePane = ({
         >
           {/* TODO: Implement the link */}
             Optional: I consent to the Trip Planner storing my historical planned trips in order to
-            improve transit services in my area. <a href={`/#${TERMS_OF_STORAGE_PATH}`} target='_blank'>More info...</a>
+            improve transit services in my area. {termsOfStorageSet && <a href={`/#${TERMS_OF_STORAGE_PATH}`} target='_blank'>More info...</a>}
         </Checkbox>
       </FormGroup>
     </div>
   )
 }
+const mapStateToProps = (state, ownProps) => {
+  return {
+    termsOfStorageSet: state.otp.config.persistence?.terms_of_storage
```

Comment:
Add a comment in `example-config.yml` to show this parameter works.

Reply:
Do we need a similar config variable for the terms of use?

Reply:
Have added an example in 62278865204a45e1afa33483a5742f794c7f5b5e. I don't think a similar config item for terms of use makes sense since otherwise the terms of use checkbox would not refer to any terms of use to accept!

Reply:
Also, there is an existing component in `ComponentContext` in which the contents for the optional Terms of Storage is defined. Maybe you can use it in this PR to check for the existence of terms of storage instead of a new config property, or combine that in another PR. Thoughts?

Reply:
I gave this some thought, having an attribute on the "dummy" components shipped within otp-rr, and then if that attribute is missing showing the link. However, I found that all our configurations ship with the dummy components anyway, so that wouldn't work unless we changed all the configurations, which I wanted to avoid.

Reply:
Can't we just check for `components.TermsOfStorage !== null`?

Reply:
Given the rush I'm inclined to accept the approach, with issue #452 to deduplicate configuration.